src/pages/nft/class/_classId/_nftId/message.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div class="bg-light-gray px-[32px] pt-[24px] pb-[48px]">
+    <NFTMessageIdentity
+      v-if="nftCollectorWalletAddress"
+      class="self-start"
+      type="collector"
+      :wallet-address="nftCollectorWalletAddress"
+      wrapper-classes="!bg-transparent"
+    />
+    <hr
+      :class="['block', 'h-[2px]', 'w-full', 'my-[16px]', 'border-shade-gray']"
+    >
+    <ul class="flex flex-col gap-[24px] w-full px-[12px]">
+      <NFTMessage
+        v-for="(m, index) in messageList"
+        :key="`${m.txHash}-${m.event}`"
+        :type="m.event"
+        :from-type="m.fromType"
+        :from-wallet="m.fromWallet"
+        :to-type="m.toType"
+        :to-wallet="m.toWallet"
+        :message="m.message"
+        :message-type="m.messageType"
+        :is-list="true"
+        :index="index"
```

Comment:
I prefer `:has-seperator="index === 0"` with default is `true`, since you don't provide this props for all `<NFTMessage/>`